index.html

```diff
@@ -689,7 +689,21 @@ <h3 class="tooling__title">
             </a>
             <span class="block smaller">"SDK Generator for REST APIs"</span>
         </h3>
-        <p class="tooling__text">Generates SDKs in various programming languages using API descriptions, such as API Blueprint. Also offers a Code-generation-as-a-Service API for CI integration.</code></p>
+        <p class="tooling__text">Generates SDKs in various programming languages using API descriptions, such as API Blueprint. Also offers a Code-generation-as-a-Service API for CI integration.</p>
+    </div></li>
+	
+	<li class="tooling__item"><div class="tooling__item--wrap">
```

Comment:
Indentation is already messed up in this file. For example see lines 782 and 788, one has 6 spaces, other has 8 spaces. There are several such issues. I don't know what are you expecting to be fixed here. Also, how to "clean" git merge history? Please give clearer instructions. Thanks.